CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to the W3C Process Document
+Thank-you for offering to contribute to the [W3C Process Document](https://www.w3.org/2017/Process-20170301/). Here we provide some details on how you can contribute. 
+
+Changes to the process document tend to fall into a few classes, and how the W3C handles these varies. Simple editorial changes may require little review, but more substantial changes will require a more substantial review and discussions by the W3C community. Please read the below, and get involved as explained. 
+
+If you have any questions, please [raise an issue](/w3c/w3process/issues).
+
+## The W3C Process Community Group
```

Comment:
I would shift this section **below** the one on simple editorial improvements. If people are making those, they don't need to know anything more except how to make a PR...

Reply:
I think an introduction giving a 'roadmap' is good practice, so I don't think I agree